src/graphics/GraphvizWiringDiagrams.jl

```diff
@@ -623,4 +624,64 @@ inverse_rank_dir(rank_dir::String) = @match rank_dir begin
   "RL" => RightToLeft
 end
 
+function to_graphviz(cp::CPortGraph; kw...)::Graphviz.Graph
+  to_graphviz(property_graph(cp; kw...))
+end
+
+""" Draw a circular port graph using Graphviz.
+
+Creates a Graphviz graph. Ports are currently not respected in the image, but
+the port index for each box can be displayed to provide clarification.
+
+# Arguments
+- `graph_name="G"`: name of Graphviz graph
+- `prog="neato"`: Graphviz program, usually "neato" or "fdp"
+- `box_labels=false`: whether to label boxes with their number
+- `port_labels=false`: whether to label ports with their number
+- `graph_attrs=Dict()`: top-level graph attributes
+- `node_attrs=Dict()`: top-level node attributes
+- `edge_attrs=Dict()`: top-level edge attributes
+
+# TODO:
+The lack of ports might be able to be resolved by introducing an extra node per
+port which is connected to its box with an edge of length 0.
+"""
+
+function property_graph(cp::CPortGraph;
```

Comment:
For consistency with existing code in this module, this function should be called `to_graphviz_property_graph`.